list-and-titles-of-products.md

```diff
@@ -0,0 +1,37 @@
+The purpose of this PR is to discuss and approve the list and correct titles of all products that we will indicate on the website in the near future.
+
+### Financial area
+
+Ubiquity Dollar 
+
+Ubiquity Card
+
+Ubiquity Payments
+
+### Management area
+
+DevPool
+
+DAO OS 
+
+DAO OS Plugins (each plugin is a product):
```

Comment:
> each plugin is a product

Yes I think that the primary SaaS monetization will be from plugins. We can promote specific plugins as recommended in any basic installation with Ubiquity OS. Like the conversation rewards seems pretty useful. 

If we wanted to, we can also hide plugins behind normal APIs so that the source code is not open. But I still want to lean towards everything open source until we have problems otherwise.